hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationServiceV1.java

```diff
@@ -110,6 +111,30 @@
         registerJavaTypeSerializers();
     }
 
+    @Override
+    public <B extends Data> B toData(Object obj, DataType type) {
```

Comment:
I found it easier to have the same journal code for OS and EE - otherwise EE would have to extend the journal implementation class and override some `toData` method and the EE code would have to instantiate this extension class. This seemed awkward.